bats/fb-content-katello.bats

```diff
@@ -172,3 +196,8 @@ EOF
   skip # because of above
   puppet agent --test && grep -q Lorem /tmp/dummy
 }
+
+#@test "Delete an Organization" {
```

Comment:
org deletion 'works', however you may get errors/warnings that prevent deletion if certain things like hostgroups are associated to the organization.